src/App.js

```diff
@@ -1,41 +1,72 @@
 import "./App.css";
 import logo from "./logo.png";
+import { app } from "firebase.config";
+import { Route, Routes } from "react-router";
+import HomePage from "pages/HomePage/HomePage";
+import Main from "Main";
+import ExplorePage from "pages/ExplorePage/ExplorePage";
+import ProfilePage from "pages/ProfilePage/ProfilePage";
+import LoginPage from "pages/LoginPage/LoginPage";
+import SignUpPage from "pages/SignUpPage/SignUpPage";
+import BookmarkPage from "pages/BookmarkPage/BookmarkPage";
+import RequiredAuth from "components/common/PrivatedRoutes/RequiredRoutes";
+import Mockman from 'mockman-js';
+import NotFoundPage from "pages/NotFoundPage/NotFoundPage";
+import { ROUTES } from "utils/routes";
+import { Suspense } from "react";
 
+console.log(app);
```

Comment:
remove console log it will slow down your render

Reply:
done